tf/orchestrator.go

```diff
@@ -68,7 +68,7 @@ func tflint() string {
 	if runtime.GOOS == "windows" {
 		return basePath + "\\tflint.exe"
 	}
-	return basePath + "/tflint"
+	return basePath + "tflint"
```

Comment:
```suggestion
	return basePath + "/tflint"
```
I think you do need a slash there...